data/mods/CRT_EXPANSION/items/crt_item_enchants.json

```
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "light_healing",
```

```suggestion
    "id": "crt_light_healing",
```
probably a good idea to keep these separate from magiclysm just in case.